src/components/error-summary/index.md.njk

```diff
@@ -36,6 +36,22 @@ There are 2 ways to use the error summary component. You can use HTML or, if you
 
 {{ example({group: "components", item: "error-summary", example: "default", html: true, nunjucks: true, open: false, size: "s", id: "default-2"}) }}
 
+### Linking from the error summary to each question
+
+You must link the errors in the error summary to the question they relate to.
+
+For questions that use a single field, for example the character count, file upload, select, textarea or text input components, link to the field.
+
+{{ example({group: "components", item: "error-summary", example: "linking", html: true, nunjucks: true, open: false, size: "s"}) }}
+
+For questions that use multiple fields, for example the date input component, you should link to the first field that contains an error. If you don't know which fields contain errors, link to the first field.
+
+{{ example({group: "components", item: "error-summary", example: "linking-multiple-fields", html: true, nunjucks: true, open: false, size: "s"}) }}
+
+For questions that use multiple inputs, such as the checkboxes or radios components, link to the first checkbox or radio.
```

Comment:
It doesn't seem quite right to me to call these inputs and the above examples 'fields' as, for example, we say 'text input'. Suggest changing to:

```suggestion
For questions that use multiple form controls, such as the checkboxes or radios components, link to the first checkbox or radio.
```

Reply:
I like the use of 'form controls' – should we change 'fields' to 'form controls' throughout?